python_modules/libraries/dagster-components/dagster_components/impls/dbt_project.py

```diff
@@ -35,14 +42,37 @@ def cli(project_path: Optional[str], init: bool) -> "DbtGenerateParams":
         return DbtGenerateParams(project_path=project_path, init=init)
 
 
+class DbtProjectComponentTranslator(DagsterDbtTranslator):
+    def __init__(
+        self,
+        *,
+        translator: Optional[DbtNodeTranslatorParams] = None,
```

Comment:
nit: maybe `translator_params` here? just feels weird to have a Translator have a `translator` property.

Reply:
agree updated